bom/pom.xml

```diff
@@ -296,6 +296,11 @@ THE SOFTWARE.
         <artifactId>j-interop</artifactId>
         <version>2.0.8-kohsuke-1</version>
       </dependency>
+      <dependency>
+        <groupId>org.kohsuke.metainf-services</groupId>
+        <artifactId>metainf-services</artifactId>
```

Comment:
Note that this will cause that library to be managed for plugin POMs (like `credentials` or `artifact-manager-s3`) for the first time. Seems like a good thing; I am not sure why it was not already in the core BOM. But if this is a problem for any reason, it could trivially be reverted, and the version fixed in the new module.

Reply:
FWIW there is an exclusion for this library elsewhere in the build: https://github.com/jenkinsci/jenkins/blob/644a261e78e9d80e2442787c8418cbdf283ac7e1/war/pom.xml#L92-L96=